website/src/styles/main.scss

```diff
@@ -4,33 +4,25 @@
 /* deprecated; use new tokens */
 @import 'dark-light-mode';
 @import 'variables';
-
 @import 'tokens-core';
 @import 'tokens-dark';
 @import 'tokens-light';
-
 @import 'components/cards';
 @import 'components/code';
 @import 'components/table';
 @import 'components/tabs';
-
 @import 'categories/design-tokens/base-tokens';
-
 @import 'layout/content';
 @import 'layout/navigation';
-
 @import 'typography';
 @import 'elements';
-
```

Comment:
Can we please keep those separators? :) I added them to separate logically differentiated styles. :)

Reply:
Fixed; `lint-staged` auto removed them.